src/main/java/uk/gov/hmcts/reform/et/helper/mapping/EmployeeObjectMapper.java

```diff
@@ -0,0 +1,67 @@
+package uk.gov.hmcts.reform.et.helper.mapping;
+
+import com.fasterxml.jackson.databind.JavaType;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.et.common.model.ccd.CaseData;
+import uk.gov.hmcts.et.common.model.ccd.items.HearingTypeItem;
+
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Converts case data that is stored in a format used in http calls and wraps it in a class for API use.
+ */
+@Slf4j
+@Service
+public final class EmployeeObjectMapper {
```

Comment:
This is a bad name which I think I originally created in et-sya-api when I didn't fully understand the project. I think this should be renamed CaseDataMapper or something similar

Reply:
Renamed